web/src/stream_data.js

```diff
@@ -848,23 +848,11 @@ export function remove_default_stream(stream_id) {
 }
 
 export function options_for_dropdown_widget() {
-    return (
-        subscribed_subs()
-            .map((stream) => ({
-                name: stream.name,
-                unique_id: stream.stream_id,
-                stream,
-            }))
-            // eslint-disable-next-line consistent-return, array-callback-return
-            .sort((a, b) => {
-                if (a.name.toLowerCase() < b.name.toLowerCase()) {
-                    return -1;
-                }
-                if (a.name.toLowerCase() > b.name.toLowerCase()) {
-                    return 1;
-                }
-                // Not possible since two streams cannot have same name.
-                // return 0;
-            })
-    );
+    return subscribed_subs()
+        .map((stream) => ({
+            name: stream.name,
+            unique_id: stream.stream_id,
+            stream,
+        }))
+        .sort((a, b) => a.name.toLowerCase().localeCompare(b.name.toLowerCase()));
```

Comment:
The `localeCompare()` suggestion by Dan was really smart, but it looks like we have our own faster way of doing this? Look at `make_strcmp()` in `util.ts`

Reply:
Also, studying the code there, you'd call it with `strcmp()`, as `make_strcmp()` is a higher-order function.

Reply:
Ah, true. What about calling `.toLowerCase()` on the arguments passed to `strcmp()`?

Reply:
Ok, made that change.